core/iwasm/fast-jit/fe/jit_emit_memory.h

```diff
@@ -44,17 +44,17 @@ bool
 jit_compile_op_f64_store(JitCompContext *cc, uint32 align, uint32 offset);
 
 bool
-jit_compile_op_memory_size(JitCompContext *cc, uint32 mem_idx);
+jit_compile_op_memory_size(JitCompContext *cc);
```

Comment:
Same as above, had better not change it, and jit_compile_op_memory_size(cc, 0) in the frontend.